pkg/providers/docker/config/template.yaml

```diff
@@ -85,13 +85,42 @@ spec:
         certSANs:
         - localhost
         - 127.0.0.1
-{{- if .extraArgs }}
         extraArgs:
-{{ .extraArgs.ToYaml | indent 10 }}
+          anonymous-auth: "false"
+          audit-policy-file: /etc/kubernetes/audit-policy.yaml
+          audit-log-path: /var/log/audit.log
+          audit-log-maxage: "30"
+          audit-log-maxbackup: "10"
+          audit-log-maxsize: "512"
+          profiling: "false"
+          request-timeout: 300s
```

Comment:
Why is this altered above the 60s default?

Reply:
I used the remediation recommended by the CIS benchmark. I read the [request timeout was made configurable](https://github.com/kubernetes/kubernetes/pull/51415) because 60s was not sufficient wait in slow network environments, so users could tune it accordingly.